mandatory/2-logic-error.js

```diff
@@ -1,16 +1,16 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
 }
```

Comment:
That's right, there is no such function in JavaScript called `wordtrim()`. You should use `trim()` instead.